### DISCUSSION TITLE
Use Google Drive/One Drive as network storage

### DISCUSSION THREAD

**Discussion by AlessandroMartinelli** [AxisCommunications/discussions#185] [General]
Hi,
I'd like to know if there is a way of configuring an AXIS camera (we use M2025-LE) in order to use to google drive or one drive account as network storage.

Thanks,
Alessandro

**Comment by vivekatoffice** [comment]
Hi @AlessandroMartinelli ,
Have you had a chance to explore potential solutions or make any progress? Your insights would be greatly appreciated.

I can only able to find the list of SMB versions supported in Axis OS:
[Network Share](https://help.axis.com/en-us/axis-os-knowledge-base#network-share)
![image](https://github.com/AxisCommunications/discussions/assets/52032125/8af69882-a738-4d5d-b5c2-4ab1fad932bc)
